src/libos/crates/io-uring-callback/src/io_handle.rs

```diff
@@ -22,10 +22,10 @@ pub enum IoState {
     Submitted,
     /// Completed with a return value.
     Completed(i32),
-    /// Cancelling (not used yet).
+    /// Cancelling.
     Cancelling,
-    /// Cancelled (not used yet).
-    Cancelled,
+    /// Cancelled with a return value
+    Cancelled(i32),
```

Comment:
The `Cancelled` variant does not need to be associated with a return value.

I have given more explanations this in my other comments about the usage of the cancel method. But one simple arguement is that even the `Cancelled` variant has an attached return value, it would have only one possible value---`-libc::ECANCELED`---according to the logic in the `complete` method. So keeping the return value for `Cancelled` is meaningless.